api/migrations/20170509230917_add_email_id_column.js

```diff
@@ -1,20 +1,8 @@
 
 exports.up = function(knex, Promise) {
     return knex.schema.table('SIP_user', table => {
-      table.string('email_id').notNullable().unique();
-    }).then(knex.select().from('SIP_user').then(users => {
-      if (process.env.OIDC_DB_ADAPTER === 'mysql') {
-        return knex.raw(`
-          UPDATE SIP_user SET email_id=LCASE(email);
-        `);
-      } else {
-        users.forEach(user => {
-          const emailID = user.email.toLowerCase();
-          knex('SIP_user').update('email_id', emailID).where('id', user.id);
-        });
-
-      }
-  }));
+        return table.string('email_id').notNull();

```

Comment:
What is the email_id field for? I see in this PR where you're doing a `WHERE` on this but I don't see any code in this PR that inserts or updates info into this column. Is this from some other PR and you're just simplifying the migration?

and if email_id is necessary, why are you removing all the code that sets `email_id` for existing users?